src/app.js

```diff
@@ -1,115 +1,78 @@
-var SVG_NS = "http://www.w3.org/2000/svg"
-
 export default function (app) {
-  var view = app.view || function () {
-    return ""
-  }
-
-  var model
+  var state //= app.state || {}
```

Comment:
Are `state` and `view` now required I am assuming? Just wondering why the hesitation 😂